docs/_docs/clustering/network-configuration.adoc

```diff
@@ -111,7 +119,7 @@ You can find the list of all properties in the javadoc:org.apache.ignite.spi.com
 [cols="1,2,1",opts="header"]
 |===
 |Property | Description| Default Value
-| `localAddress` | The local address for the communication SPI to bind to. |
+| `localAddress` | The local address for the communication SPI to bind to. Prevails over `IgniteConfiguration.localHost`. |
```

Comment:
Please update to:

_... If set, overrides `IgniteConfiguration.localHost` setting._

Reply:
Fixed.